presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -752,6 +760,8 @@ void BaseVeloxQueryConfig::updateLoadedValues(
       {QueryConfig::kMaxPartitionedOutputBufferSize,
        systemConfig->capacityPropertyAsBytesString(
            SystemConfig::kDriverMaxPagePartitioningBufferSize)},
+      {QueryConfig::kDriverCpuTimeSliceLimitMs,
+       std::to_string(systemConfig->driverCpuTimeSliceLimitMs())},
```

Comment:
shall we use folly::to\<std::string\>() ?

Also I don't quite understand this method. Why are we putting these query config into system config while initialize the system config? Thanks

Reply:
Yeah, we probably doesn't need system config. Removed.